apps/Gruntfile.js

```diff
@@ -303,7 +304,8 @@ describe('entry tests', () => {
       options: {
         // Compression currently occurs at the ../dashboard sprockets layer.
         outputStyle: 'nested',
-        includePaths: ['node_modules', '../shared/css/']
+        includePaths: ['node_modules', '../shared/css/'],
+        implementation: sass
```

Comment:
new required parameter in latest grunt-sass: https://github.com/sindresorhus/grunt-sass#usage